src/view/com/composer/text-input/TextInput.web.tsx

```diff
@@ -240,6 +241,33 @@ export const TextInput = React.forwardRef(function TextInputImpl(
     }
   }, [onEmojiInserted])
 
+  // thats fix cmd + left navigates back when the editor is focused in a @mention
+  React.useEffect(() => {
+    if (editor) {
+      const handleKeyDown = (event: KeyboardEvent) => {
+        if (
+          (event.metaKey || event.ctrlKey || event.altKey) &&
```

Comment:
```suggestion
          (event.metaKey || event.altKey) &&
```

we can omit `ctrl` event key since it does not trigger any browser navigation, which is intrisic to the bug.

Reply:
done